ckan/sample-data/upload-files.jsonl

```diff
@@ -0,0 +1,8 @@
+{"action":"resource_create","data":{"package_id":"sample-dataset-1","format":"GeoJSON","name":"海巡署臺、澎、金、馬地區各安檢所及海巡隊地址 GeoJSON 格式"},"files":{"upload":"./data/sample_data.geojson"}}
+{"action":"resource_create","data":{"package_id":"sample-dataset-1","format":"CSV","name":"海巡署臺、澎、金、馬地區各安檢所及海巡隊地址 CSV 格式"},"files":{"upload":"./data/sample_data.csv"}}
+{"action":"resource_create","data":{"package_id":"sample-dataset-1","format":"XML","name":"海巡署臺、澎、金、馬地區各安檢所及海巡隊地址 XML 格式"},"files":{"upload":"./data/sample_data.xml"}}
+{"action":"resource_create","data":{"package_id":"sample-dataset-1","format":"JSON","name":"海巡署臺、澎、金、馬地區各安檢所及海巡隊地址 JSON 格式"},"files":{"upload":"./data/sample_data.json"}}
+{"action":"resource_create","data":{"package_id":"sample-dataset-2","format":"SHP","name":"直轄市、縣市界線 (TWD97 經緯度) SHP 格式"},"files":{"upload":"./data/sample_data.zip"}}
```

Comment:
```suggestion
{"action":"resource_create","data":{"package_id":"sample-dataset-2","format":"SHP","name":"直轄市、縣市界線 (TWD97 經緯度) SHP 格式","encoding":"utf-8"},"files":{"upload":"./data/sample_data.zip"}}
```

In depositar, the `ckanext.geoview.shp_viewer.encoding` is set to `big5`, which is the encoding used by most existing Shapefile. However, this particular Shapefile is encoded in UTF-8, so its encoding must be specified explicitly.